de.dlr.sc.virsat.model.calculation.test/src/de/dlr/sc/virsat/model/calculation/compute/AEquationBuilderTest.java

```diff
@@ -115,6 +115,7 @@ public void setUp() throws Exception {
 
 	@After
 	public void tearDown() throws Exception {
+		super.tearDown();

```

Comment:
Should still be here to set the super user to false, after a test case has been executed.

Reply:
I agree, which is why I added the super.tearDown(). 
Its not removed. Its added.